### PR TITLE
Add 1909 images

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: .vsts-ci/vsts-arm-ci.yml [hunk 1]
```diff
@@ -27,10 +27,21 @@ jobs:
     preview: true
     useacr: true
 
-- template: phase.yml
+- template: .vsts-ci/phase.yml
   parameters:
-    name: centos7
```
Comment (RDIL) [inline .vsts-ci/vsts-arm-ci.yml]: So where is centos?

Comment (TravisEz13) [reply .vsts-ci/vsts-arm-ci.yml]: Centos is in the normal CI.  This was just added to verify that the acr flag was working as expected.

**Comment by TravisEz13** [comment]
the ACR CI is running here: https://dev.azure.com/powershell/docker/_build/results?buildId=38655

**Comment by RDIL** [comment]
Does `ACR`  stand for **A**zure **C**ontainer **R**egistry?

**Comment by TravisEz13** [comment]
Yes, ARC, is Azure Container Registry... They upgrade docker to support new windows versions fasters.

We are blocked waiting for ACR.
`WARNING: a Windows version 10.0.18363-based image is incompatible with a 10.0.18362 host`
https://dev.azure.com/powershell/docker/_build/results?buildId=38655&view=logs&jobId=e03d0e8e-60bc-54ba-f495-fd5edd5c5c55&taskId=18f1b78f-995d-5922-0682-6e9f4f6a64c3&lineStart=61&lineEnd=61&colStart=1&colEnd=89

**Comment by TravisEz13** [comment]
Maybe add a terms page and define that.

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 2 days

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 2 days ago

**Comment by TravisEz13** [comment]
@PoshChan Pleas remind me in 18 hours

**Comment by PoshChan** [comment]
@TravisEz13, all requests start with the magic word: `Please`

<details>
<summary>Commands available in this repo for you:
</summary>
<ul><li>
<code>retry &lt;target&gt;</code> this will attempt to retry only the failed jobs for the target pipeline, <code>restart</code> can be used in place of <code>retry</code>
</li>
<li>
<code>rebuild &lt;target&gt;</code> this will perform a complete rebuild of the target pipeline, <code>rerun</code> can be used in place of <code>rebuild</code> Supported values for &lt;target&gt; which can be a comma separated list are: <code>CI</code>
</li>
<li>
<code>get failures</code> this will attempt to get the latest failures for all of the target pipelines
</li>
<li>
<code>remind me in &lt;value&gt; &lt;units&gt;</code> this will create a reminder that will be posted after the specified duration &lt;value&gt; is a number, and &lt;units&gt; can be <code>minutes</code>, <code>hours</code>, or <code>days</code> (singular or plural)
</li>

</ul>
</details>

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 18 hours

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 18 hours ago

**Comment by TravisEz13** [comment]
@PoshChan  Please remind me in 4 days.

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 4 days. ago

**Comment by TravisEz13** [comment]
Restarted CI as ACR said their work should be done, last night.  https://dev.azure.com/powershell/docker/_build/results?buildId=39122&view=results

**Comment by kkbruce** [comment]
Any progress?

**Comment by RDIL** [comment]
@PoshChan please retry CI

**Comment by PoshChan** [comment]
@RDIL, could not find build at: , error: Invoke-RestMethod : {"$id":"1","innerException":null,"message":"The requested build 38656 could not be found.","typeName":"Microsoft.TeamFoundation.Build.WebApi.BuildNotFoundException, Microsoft.TeamFoundation.Build2.WebApi","typeKey":"BuildNotFoundException","errorCode":0,"eventId":3000}
At D:\home\site\wwwroot\Modules\AzDevOps\AzDevOps.psm1:7 char:9
+         Invoke-RestMethod -Uri $url -Authentication Basic -Credential ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (Method: GET, Reques\u2026PowerShell/6.2.3
}:HttpRequestMessage) [Invoke-RestMethod], HttpResponseException
+ FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand

**Comment by KevM** [comment]
I was looking at using the powershell image and noticed that the latest version is not published. I tried my best to follow this PR but just wanted to check in to see what was blocking the release. The PR seems to be updating the `centos7` build? 😕

**Comment by TravisEz13** [comment]
@KevM Semi-Annual Channel Images (1909 for example) are currently prioritized behind releasing PowerShell 7.0.  

CentOS has nothing to do with this change.  Complete misread of the change.

**Pull request merged**